client/src/main.ts

```diff
@@ -15,11 +15,25 @@ import router from './router'
 
 const app = createApp(App)
 
+// Object to store colour theme hexcodes
+const colourTheme = {
+  colors: {
+    primary: '#193855',
+    titles: '#D12974',
```

Comment:
Could you give these colours names which give more of an indication of the colour? For example, could the `titles`` colour be renamed to something like ``primaryPink``?

Reply:
Yup, I've changed the colour names accordingly